src/runtime/virtcontainers/sandbox.go

```diff
@@ -620,6 +621,25 @@ func newSandbox(ctx context.Context, sandboxConfig SandboxConfig, factory Factor
 		return nil, err
 	}
 
+	// If we have a confidential guest we need to cold-plug the PCIe VFIO devices
+	// until we have TDISP/IDE PCIe support.
+	coldPlugVFIO := (sandboxConfig.HypervisorConfig.ColdPlugVFIO != hv.NoPort)
```

Comment:
This doesn't check for non-confidential guests and changes the default behavior from hotplug to coldplug for non-confidential guests?

Reply:
only if config file specifies a value for `cold_plug_vfio`. the default value is `no-port`, which means "no coldplug".

Reply:
Does not change the default behaviour only if explicitly `cold_plug_vfio = <some-port>` is set, cold-plug will be executed, otherwise default hot-plug to bridge and if `hotplug_vfio_on_root_bus` is set it will hot-plug to root-port.

Reply:
Ist and additional mode off by default,  how to add devices.